.github/workflows/coveralls.yml

```diff
@@ -1,6 +1,6 @@
 name: Coveralls
 
-on: ["push", "pull_request"]
+on: ["push"]
```

Comment:
This line caused coveralls to run twice on each push. Example: https://github.com/mobxjs/mobx/pull/3603

<img width="886" alt="Screenshot 2023-01-10 at 11 50 34" src="https://user-images.githubusercontent.com/22447849/211504629-23fd69db-7343-43f9-a154-1c64c33bd91c.png">

It should be enough to run it once

Reply:
If remove `pull_request` options, PR will not trigger CI.